app/services/email_alerts_updater.rb

```diff
@@ -22,13 +38,22 @@ def unsubscribe
 
 private
 
-  def subscriber_list_slug
-    subscriber_list = Services.email_alert_api.find_subscriber_list(
-      item.subscriber_list_search_attributes,
-    )
+  def subscribers_can_be_migrated_to_successor_list?
+    successor.mapped_specialist_topic_content_id == item.content_id
+  end
+
+  def topic_subscriber_list_slug
+    params = specialist_topic_subscriber_list_params(item)
+    subscriber_list = Services.email_alert_api.find_subscriber_list(params)
     subscriber_list.dig("subscriber_list", "slug")
   end
 
+  def existing_subscriber_list_slug_for_document_collection
+    params = document_collection_subscriber_list_params(successor)
+    subscriber_list = Services.email_alert_api.find_or_create_subscriber_list(params).parsed_content["subscriber_list"]
```

Comment:
Tiny thing - do you need `.parsed_content`

eg https://github.com/alphagov/email-alert-frontend/blob/main/app/controllers/content_item_signups_controller.rb#L35-L38

Reply:
Oh good point, updated 👍